media/inf/hdc.inf

```diff
@@ -310,36 +310,42 @@ HKR, , Installer32,    0, "syssetup.dll,HdcClassInstaller"
 ; %PCI\VEN_1103&DEV_0005.DeviceDesc%=PciIde_Inst, PCI\VEN_1103&DEV_0005
 ; %PCI\VEN_1103&DEV_0008.DeviceDesc%=PciIde_Inst, PCI\VEN_1103&DEV_0008
 
-%PCI\VEN_16ca&DEV_0001.DeviceDesc%=PciIde_Inst, PCI\VEN_16ca&DEV_0001
-%PCI\VEN_1078&DEV_0102.DeviceDesc%=PciIde_Inst, PCI\VEN_1078&DEV_0102
-%PCI\VEN_1042&DEV_0102.DeviceDesc%=PciIde_Inst, PCI\VEN_1042&DEV_0102
+; %PCI\VEN_16ca&DEV_0001.DeviceDesc%=PciIde_Inst, PCI\VEN_16ca&DEV_0001
+; %PCI\VEN_1078&DEV_0102.DeviceDesc%=PciIde_Inst, PCI\VEN_1078&DEV_0102
+; %PCI\VEN_1042&DEV_0102.DeviceDesc%=PciIde_Inst, PCI\VEN_1042&DEV_0102
 
-%PCI\VEN_1283&DEV_8172.DeviceDesc%=PciIde_Inst, PCI\VEN_1283&DEV_8172
-%PCI\VEN_1283&DEV_8212.DeviceDesc%=PciIde_Inst, PCI\VEN_1283&DEV_8212
+; %PCI\VEN_1283&DEV_8172.DeviceDesc%=PciIde_Inst, PCI\VEN_1283&DEV_8172
+; %PCI\VEN_1283&DEV_8212.DeviceDesc%=PciIde_Inst, PCI\VEN_1283&DEV_8212
 
-%PCI\VEN_169c&DEV_0044.DeviceDesc%=PciIde_Inst, PCI\VEN_169c&DEV_0044
-%PCI\VEN_3388&DEV_8013.DeviceDesc%=PciIde_Inst, PCI\VEN_3388&DEV_8013
+; %PCI\VEN_169c&DEV_0044.DeviceDesc%=PciIde_Inst, PCI\VEN_169c&DEV_0044
+; %PCI\VEN_3388&DEV_8013.DeviceDesc%=PciIde_Inst, PCI\VEN_3388&DEV_8013
 
-%PCI\VEN_ffff&DEV_ffff&REV_ff.DeviceDesc%=PciIde_Inst, PCI\VEN_ffff&DEV_ffff&REV_ff
+; %PCI\VEN_ffff&DEV_ffff&REV_ff.DeviceDesc%=PciIde_Inst, PCI\VEN_ffff&DEV_ffff&REV_ff
 
 ; Compatible adapters
 %PCI\CC_0101.DeviceDesc%=PciIde_Inst,PCI\CC_0101                       ; Device ID for generic Dual PCI IDE  (UNKNOWN_PCI_IDE)
-%PCI\CC_0104.DeviceDesc%=uniata_Inst,PCI\CC_0104                       ; Device ID for generic Dual PCI IDE  (UNKNOWN_PCI_IDE)
-%PCI\CC_0105.DeviceDesc%=uniata_Inst,PCI\CC_0105                       ; Device ID for generic Dual PCI IDE  (UNKNOWN_PCI_IDE)
-%PCI\CC_0106.DeviceDesc%=uniata_Inst,PCI\CC_0106                       ; Device ID for generic Dual PCI IDE  (UNKNOWN_PCI_IDE)
-%*PNP0600.PriDeviceDesc%=uniata_Inst, Primary_IDE_Channel
-%*PNP0600.SecDeviceDesc%=uniata_Inst, Secondary_IDE_Channel
-%*PNP0600.DeviceDesc%=uniata_Inst, *PNP0600                            ; Generic_ESDI_Hard_Disk_Controller   (BAD_IDE)
-%UniATA_Name%=uniata_Inst
+%PCI\CC_0104.DeviceDesc%=PciIde_Inst,PCI\CC_0104                       ; Device ID for generic Dual PCI IDE  (UNKNOWN_PCI_IDE)
+%PCI\CC_0105.DeviceDesc%=PciIde_Inst,PCI\CC_0105                       ; Device ID for generic Dual PCI IDE  (UNKNOWN_PCI_IDE)
+%PCI\CC_0106.DeviceDesc%=PciAhci_Inst,PCI\CC_0106                      ; Device ID for generic Dual PCI IDE  (UNKNOWN_PCI_IDE)
+%*PNP0600.PriDeviceDesc%=Atapi_Inst, Primary_IDE_Channel
+%*PNP0600.SecDeviceDesc%=Atapi_Inst, Secondary_IDE_Channel
+%*PNP0600.DeviceDesc%=Atapi_Inst, *PNP0600                            ; Generic_ESDI_Hard_Disk_Controller   (BAD_IDE)
+%UniATA_Name%=PciIde_Inst
```

Comment:
Just a thought, maybe it would be better to move from global hdc.inf to an inf file within the driver module.